Storage/SQL/SQLitePasswords.swift

```diff
@@ -67,8 +91,16 @@ private class PasswordsTable<T>: GenericTable<Password> {
 
     override var factory: ((row: SDRow) -> Password)? {
         return { row -> Password in
-            let site = Site(url: row["hostname"] as? String ?? "", title: "")
-            let pw = Password(site: site, username: row["username"] as? String ?? "", password: row["password"] as? String ?? "")
+            var user = row["username"] as? String ?? ""
+            var pass = row["password"] as? String ?? ""
+
+            let encType = EncryptionType(rawValue: row["encryptionType"] as? Int ?? EncryptionType.NONE.rawValue)
+            if encType == EncryptionType.AES256 {
```

Comment:
This should be turned around to `if encType != EncryptionType.AES256` and then fail and not store anything.